README.md

```diff
@@ -60,157 +60,163 @@ Once the pipeline is initiated, you can monitor its execution using the Prefect
+|   |       |   |-- utils.py                        
 |   |       |   
-|   |       |-- visualization/
+|   |       |-- visualization/                      # Scripts to create visualizations
 |   |       |
-|   |       |-- training/
-|   |       |   |-- train_ensemble.py # some ensembles might need training
+|   |       |-- training/                           # Some ensembles might need training
+|   |       |   |-- train_ensemble.py               # Script for such potential training
 |   |       |
-|   |       |-- offline_evaluation/ # we do not have a clear routine for this yet
-|   |       |   |-- evaluate_ensemble.py 
-|   |       |   |-- evaluate_sweep.py
+|   |       |-- offline_evaluation/                 # Offline evaluation and quality assurance
+|   |       |   |-- evaluate_ensemble.py            # Script to evaluate an ensemble - can be used with calibration or test models
+|   |       |   |-- evaluate_sweep.py               # Script to run a wandb sweep - should only ever be used with the calibration partition
 |   |       |
 |   |       |-- online_evaluation/
-|   |       |   |-- evaluate_forecast.py # continues performance check
-|   |       |  
-|   |       |-- drift_detection/ # aka online evalation 
-|   |       |   |-- drift_detection_input.py # check new latest data
-|   |       |   |-- drift_detection_output.py # check new forecasts
-|   |       |   |-- drift_detection_performance.py # check new performance
+|   |       |   |-- evaluate_forecast.py            # Continuous performance check of the deployed forecasting ensemble (W&B Specific)
 |   |       |
 |   |       |-- forecasting/
-|   |           |--generate_forecast.py #script to genereate true-future fc.
+|   |           |-- generate_forecast.py            # Script to generate true-future forecasts
 |   |       
 |   |
-|   |--ensemble_002/
+|   |-- different_ensemble/                         # Next ensemble, similar structure
 |   |   |-- ...
 |   |   ...
 |   ...
 |
+|-- orchestration.py                                # Orchestration for the entire pipeline (runs all deployed models and ensembles via Prefect)
+|
+|-- documentation/                                  # All relevant documentation (wiki to come)
 |
-|-- prefect/
-|    |--scripts.py # orchestration
+|-- common_utils/                                   # Functions and classes used across multiple (but not necessarily all) models/ensembles  
+|       |-- stepshifter.py                          # Updated stepshifter function
+|       |-- set_paths.py                            # Sets all paths for imports, data, utils etc. Machine invariant.
+|       |-- get_data.py                             # General function that takes the general get_partition and a model specific config_feature_set.py to fetch model specific data
+|       |-- get_partition.py                        # Get data partitions for splits pertaining to validation, testing, and forecasting
```

Comment:
Shouldn't this be in a config file instead, since we are just changing parameters?

Reply:
Jeps - actually did that at one point bc it made sense, but then realized that was not what we agreed upon so changed it to common_utills. However, I've now made a common_config dir in root and put it there - as we both seems to agree that make more sense